x-pack/plugins/security_solution/cypress/tasks/create_new_rule.ts

```diff
@@ -251,6 +278,17 @@ export const selectThresholdRuleType = () => {
   cy.get(THRESHOLD_TYPE).click({ force: true });
 };
 
+export const waitForTheRuleToBeExecuted = () => {
+  cy.get('[data-test-subj=ruleStatus]')
+    .invoke('text')
+    .then((ruleStatus) => {
+      if (ruleStatus !== 'succeeded') {
+        cy.get('[data-test-subj=refreshButton]').click();
```

Comment:
Without the commented-out assertion below, there's nothing to synchronize the results of the refresh, right? Here's how I'm picturing it:

1. page is loaded, rule has not finished executing
2. we click refresh
3. We continue on with the test, with no guarantee that the rule has executed

Reply:
I'll take a deeper look to this.